### ISSUE TITLE
No RateLimit present in response

### ISSUE THREAD

**Issue by glensc** [phar-io/phive#252]
```
$ phive install eventum/cli
Phive 0.13.2 - Copyright (C) 2015-2020 by Arne Blankerts, Sebastian Heuer and Contributors
[ERROR]   No RateLimit present in response
```

**Comment by glensc** [comment]
also, no convenient way to lookup options for install command:

```
$ phive install --help
Phive 0.13.2 - Copyright (C) 2015-2020 by Arne Blankerts, Sebastian Heuer and Contributors
[ERROR]   Error while processing arguments to command 'install':
[ERROR]   Unknown option: help
```

and `phive help` command is not intuitive, and it's too long to read, have to filter through pager to able to read it top to bottom.

**Comment by theseer** [comment]
Duplicate of #228

**Comment by glensc** [comment]
The duplicate issue is closed with no resolution. The problem is still present.

Either of the issues should be re-opened until the fix is released. I, for example, did not find the problem being reported, as it was closed issue and I installed the latest available version (0.13.2).

- https://github.com/phar-io/phive/issues?q=is%3Aissue+sort%3Aupdated-desc+No+RateLimit+present+in+response+is%3Aopen

**Comment by theseer** [comment]
The problem is solved. The fact it hasn't been released yet is unfortunate but not a reason to keep this bug open. Sorry about that.
